boot/src/main/java/io/edpn/backend/application/controller/BootTradeModuleController.java

```diff
@@ -1,14 +1,15 @@
 package io.edpn.backend.application.controller;
 
 import io.edpn.backend.trade.application.controller.v1.DefaultTradeModuleController;
-import io.edpn.backend.trade.domain.service.v1.BestCommodityPriceService;
+import io.edpn.backend.trade.domain.service.v1.FindCommodityMarketInfoService;
+import io.edpn.backend.trade.domain.service.v1.LocateCommodityService;
 import org.springframework.web.bind.annotation.RestController;
 
 
 @RestController
```

Comment:
Overall, I have no objection with this code, but wanted to ask first, no `@RequestMapping` annotation anywhere? Have I missed it?

I see the test classes etc, but these use the Controller implementation directly.

I honestly forget the "mount point" for a RestController without RequestMapping, but if you've seen it, I'm happy to merge this. If we don't have a RequestMapping, then front end can't use it, regardless of how much data we might not have.